src/backbone.collectionView.js

```diff
@@ -774,7 +782,7 @@
 				axis : "y",
 				distance : 10,
 				forcePlaceholderSize : true,
-				items : this._isRenderedAsTable() ? "> tbody > tr:not(.not-sortable)" : "> li:not(.not-sortable)",
+				items : this._isRenderedAsTable() ? "> tr:not(.not-sortable)" : "> li:not(.not-sortable)",
```

Comment:
Maybe you did not see the additional lines of the commit message of [66a6b3a](https://github.com/fvgoto/backbone.collectionView/commit/66a6b3ac0e48bbd24818dc44c0b1ddf15b3695f5) or I was not clear enough:
By making `tbody` instead of `table` to be `sortable`, as is stated in [jQuery docs (italic Note just below QuickNav index](http://api.jqueryui.com/sortable/)), the `> tbody` part falls out of the `items:` option selector. That's why the two bits are in the same commit.

The distinction if we're in list or in tabular context an if we have `tbody` or not is done in your `this._getContainerEl()`, which I proposed to use at that spot.
(NB: I'm quite sure we could find some strange browser (probably called "IE") that does not insert `tbody`... )

So I don't see anything wrong with this commit.